src/Forms/Form.php

```diff
@@ -1547,10 +1547,8 @@ public function getData()
      *
      * This is returned when you access a form as $FormObject rather
      * than <% with FormObject %>
-     *
-     * @return DBHTMLText
      */
-    public function forTemplate()
+    public function forTemplate(): string
```

Comment:
These should all return an explicit string. This is the raw HTML that will be directly input into the parent template.